collect_app/src/main/java/org/odk/collect/android/widgets/utilities/GeoWidgetUtils.kt

```diff
@@ -0,0 +1,132 @@
+package org.odk.collect.android.widgets.utilities
+
+import android.content.Context
+import android.location.Location
+import org.odk.collect.android.R
+import org.odk.collect.maps.MapPoint
+import org.odk.collect.shared.strings.StringUtils.removeEnd
+import timber.log.Timber
+import java.text.DecimalFormat
+import kotlin.math.abs
+import kotlin.math.absoluteValue
+
+object GeoWidgetUtils {
+
+    @JvmStatic
+    fun getGeoPointAnswerToDisplay(context: Context, answer: String?): String {
+        try {
+            if (answer != null && answer.isNotEmpty()) {
+                val parts = answer.split(" ".toRegex()).toTypedArray()
```

Comment:
What's the point of that `toRegex()`?

Reply:
No idea! I do remember thinking I should experiment with taking it out and then never tried. Will do that.

Reply:
You have removed some of them but there are more in GeoWidgetUtils for example (and maybe in other classes). Could you remove those calls too?

Reply:
There is one more call in GeoWidgetUtils and in its test.